src/main.css

```diff
@@ -0,0 +1,196 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

Comment:
Only `box-sizing` property allows for `*` selector. Move other properties to html or body tag